test/parallel/test-repl-missing-parenthesis-recoverable.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+/*
```

Comment:
We've got block comments all through the codebase. However, for a single line comment, it seems unnecessary. Either way, not a deal breaker.